southpark-search/pods/text_loader.py

```diff
@@ -3,10 +3,10 @@
 
 from typing import Dict
 
-from jina.executors.crafters import BaseDocCrafter
+from jina.executors.crafters import BaseCrafter
 
 
-class TextExtractor(BaseDocCrafter):
+class TextExtractor(BaseCrafter):
     def craft(self, text: str, *args, **kwargs) -> Dict:
         *_, s = text.split('[SEP]')
         return dict(weight=1., text=s, meta_info=text.encode('utf8'))
```

Comment:
is this the reason we are extracting `meta_info` when printing `top_k`? I see it is. I would prefer to have the `character name` in the `meta_info`. I think it can lead to missunderstanding, but since it is working, you can just ignore my previous comments.

Reply:
That makes sense. But considering the limited time of migrating, maybe I'll refactor it later